qa/test_framework/util.py

```diff
@@ -75,13 +75,14 @@ def initialize_datadir(dirname, n):
     datadir = os.path.join(dirname, "node"+str(n))
     if not os.path.isdir(datadir):
         os.makedirs(datadir)
-    with open(os.path.join(datadir, "zcash.conf"), 'w') as f:
+    with open(os.path.join(datadir, "zen.conf"), 'w') as f:
         f.write("regtest=1\n")
         f.write("showmetrics=0\n")
         f.write("rpcuser=rt\n")
         f.write("rpcpassword=rt\n")
         f.write("port="+str(p2p_port(n))+"\n")
         f.write("rpcport="+str(rpc_port(n))+"\n")
+        f.write("websocket=1\n")
```

Comment:
In case if we have more than 1 MC node with a websocket enabled we should specify different ports for them.
Such case is very useful for us, when we will start test mc fork impact on sc network, so different SC nodes will be connected to different mc nodes.

Reply:
ok, good

Reply:
please, put it as a different task, and add the test when you'll have time for it.

Reply:
Done